### PR TITLE
feat: add pagertree provider

### LINKED ISSUE #1127: pagertree provider
https://pagertree.com/

### PR DESCRIPTION
**Pull request by 35C4n0r**
- Used pagertree REST API to add agertree provider.
- Closes #1127 
- /claim #1127

Closes # 

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: docs/mint.json [hunk 1]
```diff
@@ -82,47 +82,53 @@
         {
```
Comment (shahargl) [inline docs/mint.json:82]: why all of these changes?

Comment (35C4n0r) [reply docs/mint.json:82]: Removed formatting, addded the providers I've added till now, and sorted them

**Comment by shahargl** [comment]
Can you change the logo to transparent bg?

**Comment by 35C4n0r** [comment]
> Can you change the logo to transparent bg?

@shahargl done!

**Comment by shahargl** [comment]
merged! @35C4n0r

**Pull request merged**